src/FSharp.Stats.Interactive/test.ipynb

```diff
@@ -26,7 +26,7 @@
     {
      "data": {
       "text/html": [
-       "<div><div><strong>Restore sources</strong><ul><li><span> C:/Users/schne/source/repos/fslaborg/FSharp.Stats/pkg</span></li></ul></div><div></div><div><strong>Installed Packages</strong><ul><li><span>FSharp.Stats, 0.0.0-dev</span></li><li><span>FSharp.Stats.Interactive, 0.0.0-dev</span></li></ul></div></div>"
+       "<div><div><strong>Restore sources</strong><ul><li><span> C:/Users/schne/source/repos/fslaborg/FSharp.Stats/pkg</span></li></ul></div><div></div><div><strong>Installed Packages</strong><ul><li><span>FSharp.Stats, 0.0.1-dev</span></li><li><span>FSharp.Stats.Interactive, 0.0.1-dev</span></li></ul></div></div>"
```

Comment:
In L11 you wrote "The version of the package is always `0.0.0-dev`.", but here you changed it to 0.0.1-dev. Is that intended?

Reply:
fixed